### PR TITLE
[full][gecko] Move rustc bash-completion fix from gecko to full image

### PR DESCRIPTION
**Pull request by jankeromnes**

### CODE REVIEW

**Comment by akosyakov** [comment]
I still see command not found, although bashrc seems to have changes:
Screenshot 2020-11-23 at 08 44 58

**Review by akosyakov — APPROVED** [review]
It works now:
Screenshot 2020-11-23 at 10 16 32

File: full/Dockerfile [hunk 1]
```diff
@@ -270,7 +270,10 @@ RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
     && .cargo/bin/rustup completions bash | sudo tee /etc/bash_completion.d/rustup.bash-completion > /dev/null \
     && .cargo/bin/rustup completions bash cargo | sudo tee /etc/bash_completion.d/rustup.cargo-bash-completion > /dev/null \
     && .cargo/bin/rustup target add x86_64-unknown-linux-musl \
-    && grep -v -F -x -f /home/gitpod/.profile_orig /home/gitpod/.profile > /home/gitpod/.bashrc.d/80-rust
+    && grep -v -F -x -f /home/gitpod/.profile_orig /home/gitpod/.profile > /home/gitpod/.bashrc.d/80-rust \
+    # Inject the path update to the top as bash completion needs to call rustc
+    && sed -i '1s|^|export PATH=\"\$PATH:$HOME/.cargo/bin\"\n\n|' $HOME/.bashrc
```
Comment (akosyakov) [inline full/Dockerfile:275]: so maybe it is not necessary? 🤷‍♂️

**Comment by karlloic** [comment]
@akosyakov Can this be added to the `workspace-postures` image as well? Thanks

**Comment by akosyakov** [comment]
@karlloic Don't know, please try to file an issue or a PR someone more knowledgeable will answer.

**Comment by jankeromnes** [comment]
@karlloic `gitpod/workspace-postgres` is actually [based on](https://github.com/gitpod-io/workspace-images/blob/4cb00b4f2a8f8d6a1642b83ec75179c7dc57fca6/postgres/Dockerfile#L1) `gitpod/workspace-full`, so it should already have this change.

Are you still seeing errors? If so, please share your repository URL and screenshots of the errors you see (e.g. by posting in https://community.gitpod.io )

**Comment by karlloic** [comment]
@jankeromnes Yes, I am stiff facing this error, as of two days ago. I will create an issue and share more info + screenshots. Thanks.

**Comment by karlloic** [comment]
@jankeromnes Strangely enough, the problem just vanished. If I run into it again, I will create an issue. Thanks

**Pull request merged**